common/djangoapps/util/organizations_helpers.py

```diff
@@ -36,6 +36,20 @@ def get_organization(organization_id):
     return organizations_api.get_organization(organization_id)
 
 
+def get_organization_by_short_name(organization_short_name):
+    """
+    Client API operation adapter/wrapper
+    """
+    if not settings.FEATURES.get('ORGANIZATIONS_APP', False):
```

Comment:
we should use the new organizations_enabled function instead of this check here -- in addition we could update the other ORGANIZATIONS_APP checks throughout the code